README.md

```diff
@@ -13,81 +13,25 @@ displays the contacts themselves, fetching them from content store:
 
 ## Technical documentation
 
-This is a Ruby on Rails application that provides a private admin UI for signon users with permission to allow them to manage the contacts for their organisation.
+This is a Ruby on Rails app, and should follow [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html).
 
-The app has its own database (MySQL), which is used by the admin UI. This is in contrast to the separate [government-frontend](https://github.com/alphagov/government-frontend) app which reads contacts from the [content-store](https://github.com/alphagov/content-store). The admin UI part of this app handles publishing the contacts to the [publishing-api](https://github.com/alphagov/publishing-api) so that they are present for [finder-frontend](https://github.com/alphagov/finder-frontend) and [government-frontend](https://github.com/alphagov/government-frontend) to read.
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) to run the application and its tests with all the necessary dependencies. Follow [the usage instructions](https://github.com/alphagov/govuk-docker#usage) to get started.
 
-### Dependencies
+**Use GOV.UK Docker to run any commands that follow.**
 
-* Ruby 2.3.0
-* MySQL
-* [whitehall](https://github.com/alphagov/whitehall)
-  1. to access the organisations API and maintain parity with the organisations managed in whitehall.
-  2. to access the world locations API to provide the correct list of countries for the country dropdown in the address form
-* [signon](https://github.com/alphagov/signonotron2) - for managing user authentication
-* [content-store](https://github.com/alphagov/content-store) - for doing some
-  command-line based tasks to withdraw and redirect a contact
+### Before running ths app
 
-### Running with govuk-docker
-
-Setup:
-
-```sh
-govuk-docker-run env RAILS_ENV=test env bin/rake db:create
-```
-
-Then run tests:
+The best way to get a database with good seed data is to replicate from Integration. Alternatibely, you can use a combination of import commands and old seed data to seed the database.
 
 ```sh
-govuk-docker-run env RUNNING_IN_CI='true' bundle exec rspec
-```
-
-#### Previewing the application
-
-Seed the database, with ENV overrides so that `ImportOrganisations` doesn't attempt to fetch organisations from dev:
-
-```sh
-govuk-docker-run env GOVUK_APP_DOMAIN='https://www.gov.uk' env GOVUK_WEBSITE_ROOT='https://www.gov.uk' bin/rake db:seed
-```
-
-Run the app:
-
-```sh
-govuk-docker-up
+env GOVUK_APP_DOMAIN='https://www.gov.uk' env GOVUK_WEBSITE_ROOT='https://www.gov.uk' bin/rake db:seed
```

Comment:
I know the idea is to avoid using `govuk-docker-run` throughout the README, but in this case I think it's worth keeping the prefix in, as `env` is a Docker CLI argument so makes no sense in isolation.

Reply:
So [`env`](https://en.wikipedia.org/wiki/Env) is actually a normal UNIX command. We have to use it with Docker since other alternatives don't work:

```
VAR=value govuk-docker-run ... # env var not passed through

govuk-docker-run VAR=value ... # error, not a command
```

`env` is nice because it works with and without GOV.UK Docker, so I think it's reasonable to keep here.

Reply:
Oops - I really should have Googled that first 😄  I haven't seen `env` used outside of Docker commands. Thanks for the explanation!